.github/workflows/build-test.yml

```diff
@@ -29,8 +29,7 @@ jobs:
         with:
           path: ${{ steps.yarn-cache-dir.outputs.YARN_CACHE_DIR }}
           key: yarn-cache-${{ runner.os }}-${{ steps.yarn-version.outputs.YARN_VERSION }}-${{ hashFiles('yarn.lock') }}
-      - run: yarn --frozen-lockfile
```

Comment:
According to [migration guide](https://yarnpkg.com/migration/guide).
the renaming of --frozen-lockfile into --immutable